### DISCUSSION TITLE
Not sure if example code is a typo or intended.

### DISCUSSION THREAD

**Discussion by firstimedeveloper** [microsoft/ML-For-Beginners#101] [Q&A] [answered]
I was wondering if the following code 
```
from sklearn.preprocessing import LabelEncoder

new_pumpkins.iloc[:, 0:-1] = new_pumpkins.iloc[:, 0:-1].apply(LabelEncoder().fit_transform)
new_pumpkins.iloc[:, 0:-1] = new_pumpkins.iloc[:, 0:-1].apply(LabelEncoder().fit_transform)
```
seen in [2-Regression](https://github.com/microsoft/ML-For-Beginners/tree/main/2-Regression/3-Linear#prepare-your-data-for-regression) under the header, "Prepare your data for regression" is written twice as intended or is a typo.

Posting this as a discussion instead of a PR as I'm not sure myself. Thanks to anyone in advance!

**Comment by vishvanathan-k** [comment, marked as answer]
I guess it a typo, because it does the same task twice which is unnecessary as far as i know .
Edit: I have also opened a PR #102

**Reply by jlooper** [reply]
thanks for the fix, looks like I messed it up :)

**Reply by vishvanathan-k** [reply]
@jlooper no problem :)
